### PR TITLE
feat(VisuallyHidden): Add VisuallyHidden component

### PR DESCRIPTION
**Pull request by thaytharma**
Added `VisuallyHidden`-component

I started with using `dnb-sr-only`-class, but ended up using the the`srOnly`-mixin instead as I felt it was a "cleaner".
I also added a `focusable`-option which I saw being used in other libraries like in [Bootstrap](https://getbootstrap.com/docs/5.0/helpers/visually-hidden/)

I'm open to suggestions as this is an initial version of the component 😊

### CODE REVIEW

File: packages/dnb-eufemia/src/components/visually-hidden/VisuallyHidden.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+   * Hide an element by default, but to display it when it’s focused (e.g. by a keyboard-only user)
+   * Default: false
+   */
+  focusable?: boolean
+
+  /**
+   * Root element of the component
+   * Default: div
+   */
+  element?: string
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/visually-hidden/VisuallyHidden.tsx:31]: Maybe use ReactNode here? This way its even possible to send in an existing component.


```jsx
const Component = () => <div>...</div>
render(<VisuallyHidden element={Component}>hidden content</VisuallyHidden>)
```

**Review by tujoworker — APPROVED** [review]
This is awesome! Fantastic work ✨

File: packages/dnb-eufemia/src/components/visually-hidden/VisuallyHidden.tsx (hunk 1 above)
Comment (thaytharma) [reply packages/dnb-eufemia/src/components/visually-hidden/VisuallyHidden.tsx]: Great suggestion!
This has been updated 🚀

**Pull request merged**